Prevention Act;Prevents sandmining, scamming and trapping.html

```diff
@@ -12,7 +12,7 @@ <h3>Scams<h3>
 <p>
 <h3>Trapping<h3>
 <p>
-    If a player trap a citizen of Sudan they will be required to give them their stuff
-    back and if they have a high ranking position in a town/nation it may be 
+    If a player traps a citizen of Sudan they will be required to give them their stuff
+    back and if they have a high ranking position such as military member or minister in the nation it may be 
```

Comment:
Slight grammar change and clarity required, see the suggestion below.
```suggestion
    If a player traps a citizen of Sudan, they will be required to give the victim their stuff
    back. If they have a high ranking position such as military member or minister in the nation it may be 
```